From 1f679da30a33f3ddad14bc6b2be0795160ae12b8 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Thu, 9 Apr 2015 10:19:13 -0400
Subject: [PATCH] include: Fix endianness setup

Need to make sure X_{BIG,LITTLE}_ENDIAN actually get defined

Signed-off-by: Adam Jackson <ajax@redhat.com>
---
 include/dix-config.h.in  |    1 +
 include/xorg-server.h.in |    1 +
 2 files changed, 2 insertions(+)

Index: a/include/dix-config.h.in
===================================================================
--- a.orig/include/dix-config.h.in
+++ a/include/dix-config.h.in
@@ -499,6 +499,7 @@
 
 /* byte order */
 #undef X_BYTE_ORDER
+#include <X11/Xarch.h>
 
 /* Listen on TCP socket */
 #undef LISTEN_TCP
Index: a/include/xorg-server.h.in
===================================================================
--- a.orig/include/xorg-server.h.in
+++ a/include/xorg-server.h.in
@@ -233,5 +233,6 @@
 
 /* byte order */
 #undef X_BYTE_ORDER
+#include <X11/Xarch.h>
 
 #endif /* _XORG_SERVER_H_ */
